packages/cli/src/lib/pwaValidationHelper.ts

```diff
@@ -11,7 +11,8 @@ function getColor(score: ScoreResult): string {
   }
 }
 
-export function printValidationResult(validationResult: PwaValidationResult, log: Log): void {
+export function printValidationResult(validationResult: PwaValidationResult, log: ConsoleLog):
+ void {
```

Comment:
There's not need to declare `void` return types. TypeScript will infer it.

Reply:
it throws an error when I remove it.

Reply:
Yeah, I checked other places. It seems we have a rule around that.